src/contracts/strategies/StrategyWrapper.sol

```diff
@@ -1,152 +0,0 @@
-// SPDX-License-Identifier: BUSL-1.1
-pragma solidity =0.8.12;
```

Comment:
any big reason for deleting this file? I could see situations in which it is preferable to use

Reply:
Agreed offline it's OK to delete